src/documentation/transxchange_read.md

```diff
@@ -92,8 +92,17 @@ tram | Tramway | Tramway| Tramway| Tramway
 trolleyBus | Shuttle | Shuttle | Shuttle | Shuttle
 
 ### routes.txt
-A Route is created from a line and a direction of journey patterns (*StandardService/JourneyPattern/Direction*/).
-Routes might not be specified at all in the xml --> reconstruct routes after trips?
+A Route is created from a line and a direction of journey patterns (*StandardService/JourneyPattern/Direction*).
+
+NTFS field | TransXChange element | Mapping rule/Comment
+--- | --- | ---
+route_id | *Services/Service/ServiceCode*, *StandardService/JourneyPattern/Direction* | This field is prefixed and formed by the concatenation of *Services/Service/ServiceCode* and *StandardService/JourneyPattern/Direction* separated by a `:`. Ex. "\<prefix>:1_58_BC:inbound".
```

Comment:
Do you think it would be correct to specify this as
```suggestion
route_id | line_id, *StandardService/JourneyPattern/Direction* | This field is prefixed and formed by the concatenation of the line ID and *StandardService/JourneyPattern/Direction* separated by a `:`. Ex. "\<prefix>:1_58_BC:inbound".
```
It feels that if we ever change what is going to define what is a line ID, then we're gonna need to adapt the route ID... unless we define them already based on the line ID. Would it be correct to say that?

Reply:
`line_id` is more related to the implementation. I would leave `Services/Service/ServiceCode` to not mix things coming from NTFS and transxchange

Reply:
We discussed it with @papailio and `line_id` is not gonna be used.